app/src/main/java/org/oppia/android/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -113,6 +119,23 @@ class AudioFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun startSpotlights() {
+    val audioLanguageIconSpotlightTarget = SpotlightTarget(
+      binding.audioLanguageIcon,
+      resourceHandler.getStringInLocale(R.string.voiceover_language_icon_spotlight_hint),
+      SpotlightShape.Circle,
+      Spotlight.FeatureCase.VOICEOVER_LANGUAGE_ICON
+    )
+
+    getSpotlightFragment().requestSpotlightViewWithDelayedLayout(audioLanguageIconSpotlightTarget)
+  }
+
+  private fun getSpotlightFragment(): SpotlightFragment {
+    return activity.supportFragmentManager.findFragmentByTag(
+      SpotlightManager.SPOTLIGHT_FRAGMENT_TAG
+    ) as SpotlightFragment
```

Comment:
Ditto here & everywhere else in the PR.

Reply:
here as well? I'm not sure but it seems this place shouldnt be changed.

Reply:
Here as well, since findFragmentByTag can return null (or the wrong fragment).